File: Java/Bubblesort.java
class Program {
  public static int[] bubbleSort(int[] array) {
    // Write your code here.
		for(int i=0;i<array.length;++i){
		for(int j = i+1;j<array.length;++j){
			if(array[i]>array[j]){
				int temp = array[i];
				array[i]=array[j];
				array[j]=temp;
			}
		}
	}
  return array;
    
  }
}

File: Python/bubblesort.py
def bubbleSort(array):
    # Write your code here.
	for i in range (len(array)):
		for j in range(i+1, len(array)):
			if array[i] > array[j]:
				temp = array[i]
				array[i]=array[j]
				array[j]=temp
	return array			
    pass
n = int(input())
arr = list(map(int, input().strip().split()))
arr1 = bubbleSort(arr)
for i in arr1:
	print(arr1, end = " ")
